src/Bathymetry.jl

```diff
@@ -22,14 +29,38 @@ using Printf
 Regrid bathymetry associated with the NetCDF file at `path = joinpath(dir, filename)` to `target_grid`.
 If `path` does not exist, then a download is attempted from `joinpath(url, filename)`.
 
-TODO: describe keyword arguments.
+Arguments:
+==========
+
+- target_grid: grid to interpolate onto
+
+Keyword Arguments:
+==================
+
+- height_above_water: limits the maximum height of above-water topography (where h > 0). If
+                      `nothing` the original topography is retained
+
+- minimum_depth: minimum depth for the shallow regions. `h > minimum_depth` will be considered land
+
+- dir: directory of the bathymetry-containing file
+
+- filename: file containing bathymetric data. Must be netcdf with fields:
+            (1) `lat` vector of latitude nodes
+            (2) `lon` vector of longitude nodes
+            (3) `z` matrix of depth values
+
+- interpolation_passes: regridding/interpolation passes. The bathymetry is interpolated in
+                        `interpolation_passes - 1` intermediate steps. With more steps the 
+                        final bathymetry will be smoother.
```

Comment:
Can you add more to this description? I understand that it smooths things out but I don't understand how it works.